### ISSUE TITLE
Feature request to allow DDNS 

### ISSUE THREAD

**Issue by rusa99ec** [Florianisme/WakeOnLan#5]
Could it be possible to allow text in the "Broadcast address"? Some people with access to a public IP address are using services like duckdns or no-ip in order to keep their public IP always updated, for external WOL.
Thanks in advance, love this app. It just works and is easy to use.

**Comment by Florianisme** [comment]
Hi, do you really mean the Broardcast Address? That is the address that the Magic Packet get's sent to. 
I used a VPN connection to my home network when I was away from home to turn on my devices.

**Comment by robbie99ec** [comment]
Yep, I really mean the broadcast address. I.E. instead of writing my public IP I would use something like "mypersonalserver.duckdns.org". You may ask "well just type your public IP it doesnt matter", most ISPs dont give you a static IP so the public IP changes sometimes like after restarting your router or losing electricity.

**Comment by Florianisme** [comment]
Will be part of the new beta release 1.7.2 which I will publish in a few minutes

**Comment by ShardYaco** [comment]
Where can I find that release? I have installed the latest 1.7.3. but it seems that DNS resolving is not supported there. Still requires inputting IP address.

**Comment by Florianisme** [comment]
You'll have to to join the beta. It should be 1.7.2 as I had to release 1.7.3 as a Hotfix in production :)

**Comment by Florianisme** [comment]
Released to production in 1.8.0
